clang/tools/clang-offload-packager/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ add_clang_tool(clang-offload-packager
   ${tablegen_deps}
   )
 
+setup_host_tool(clang-offload-packager CLANG_OFFLOAD_PACKAGER_EXE
```

Comment:
I wonder if this and `llvm-ar` changes should be upstreamed right away

Reply:
I considered including this upstream but there is nothing there that would make use of it, it's not currently an issue there, it's only an issue in DPC++. But it also wouldn't do any harm in upstream LLVM, so if you like, I'm happy to submit it and see how they respond.

Reply:
I don't have a strong opinion here, but it probably worth a try considering our general upstreaming efforts, i.e. it will be a problem in the upstream as well at some point - perhaps you can use that as a motivation for the change.

Reply:
Spotted a typo while copying over the changes that made sense for upstream LLVM, now fixed, so that's something good to come out of it regardless :)

Reply:
https://github.com/llvm/llvm-project/pull/118635